src/components/tooltip/daikin-tooltip.ts

```diff
@@ -190,61 +180,84 @@ export class DaikinTooltip extends LitElement {
     this._autoUpdateCleanup = null;
   }
 
-  private _changeOpenState(state: boolean) {
-    if (this.open === state) {
-      return;
+  private _changeOpenState() {
+    this._popover.togglePopover(!this.open);
+
+    this.open = !this.open;
+  }
```

Comment:
It is easier to understand the function that receives the value than the toggle.

```suggestion
  private _setOpenState(newState: boolean): void {
    if (this.open === newState) {
      return;
    }

    this._popover.togglePopover(newState);
    this.open = newState;
  }
```